src/lib/system/system.service.spec.ts

```diff
@@ -123,6 +123,42 @@ describe('System Service:', () => {
     ),
   ));
 
+  it('expect to do a health request succesfully', async(
```

Comment:
Let's revert these tests from this PR, since they are unrelated and I do not think we need coveralls to pass to merge in PRs.

Reply:
will do. ill save it for another pr. was just to get coveralls to stop yelling at me lol

Reply:
done.